gammapy/scripts/config/schema.yaml

```diff
@@ -186,6 +186,57 @@ properties:
             dataset-type:
                 enum: [not assigned, SpectrumDatasetOnOff, MapDataset]
             containment_correction: {type: boolean}
+            offset-max: {type: string}
+            geom:
+                type: object
+                additionalProperties: false
+                properties:
+                    binsz: {type: number}
+                    coordsys:
+                        type: string
+                        enum: [CEL, GAL]
+                    proj: {type: string}
+                    skydir:
+                        type: array
+                        items: {type: number}
+                        minItems: 2
+                        maxItems: 2
+                    width:
+                        type: array
+                        items: {type: number}
+                        minItems: 1
+                        maxItems: 2
+                    axes:
+                        type: array
+                        additionalProperties: false
+                        items:
+                            "$ref": "#/definitions/map_axis"
+                required: [axes]
+            geom-irf:
+                type: object
+                additionalProperties: false
+                properties:
+                    binsz: {type: number}
+                    coordsys:
+                        type: string
+                        enum: [CEL, GAL]
+                    proj: {type: string}
+                    skydir:
+                        type: array
+                        items: {type: number}
+                        minItems: 2
+                        maxItems: 2
+                    width:
+                        type: array
+                        items: {type: number}
+                        minItems: 1
+                        maxItems: 2
+                    axes:
+                        type: array
+                        additionalProperties: false
+                        items:
+                            "$ref": "#/definitions/map_axis"
+                required: [axes]
```

Comment:
I think we can extract the two duplicated geom sections and reference to them as definitions, in the same way it is done with i.e. models or map-axis

Reply:
Done